app/src/main/java/com/kickstarter/ui/views/compose/search/SearchTopBar.kt

```diff
@@ -59,17 +53,20 @@ fun SearchTopBarPreview() {
         SearchTopBar(
             onBackPressed = {},
             onValueChanged = {},
+            selectedFilterCounts = mapOf(),
```

Comment:
You might wanna initialize the map here with values similar to the Screen 
mapOf(
        FilterRowPillType.SORT.name to 0,
        FilterRowPillType.CATEGORY.name to 0
    )

the compose preview is failing and the design system screen crashes due to missing key SORT
<img width="1581" alt="Screenshot 2025-03-12 at 10 48 02 AM" src="https://github.com/user-attachments/assets/49d305dd-f535-4e36-8f76-e3989d16e285" />